CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Release Date: _unreleased_
 - Return components effective role assignments only for assigned modules.
 - Dropped internal `balance_changes` fallback mechanism. As of right now this information is ingested as part of regular transaction ingestion process.
 - Reworked internal mechanism used to fetch network configuration. Is no longer stored in the underlying database and it is shared across all services.
+- Fixed `state_version`-based ledger state `at_ledger_state`/`from_ledger_state` constraints which could result in inaccurate lookups previously.  
```

Comment:
can you please explain impact on users? Right now it's not even clear for me what has changed i.e 

if querying `at_ledger_state`/`from_ledger_state`:
- using `state_version` is now strict, if certain state version doesn't exist in gateway database, we're gonna (throw/return empty result?)
- using `timestamp` - extended validation  (explain what has changed)

Reply:
1. Extended changelog with fix explanation.
2. There's no change to `timestamp`-based lookups. I've change method name to reduce potential confusion.